### PR TITLE
iOS Control Rate feature. (#42)

### PR DESCRIPTION
**Pull request by Amanida**
Completed iOS Control Rate feature. (issue #42 )
- Added `UISlider` to control rate
- Added `rate` property to `Speaker`.
- Added `rateLabel` to show rate value.

p.s - In this version, it shows its rate value`from 0.0 to 1.0.` I think there's some better idea than mine. If you have, please reply it.

### CODE REVIEW

File: iOS/ViewController.swift [hunk 1]
```diff
@@ -69,6 +70,12 @@ class ViewController: UIViewController, SpeakerDelegate, UIPickerViewDelegate, U
         present(alertController, animated: true, completion: nil)
     }
     
+    @IBAction func rateChanged(_ sender: UISlider) {
+        let value = roundf(100 * sender.value) / 100
```
Comment (eeneguee) [inline iOS/ViewController.swift:74]: When to display the quickness, natural number expression is usually used.
How about divide the speed level about 10 steps?

Comment (Amanida) [reply iOS/ViewController.swift:74]: Thank you for reviewing @eeneguee !. I tried to divided it into 10 levels but it didn't look natural... 😢 The rate was changed too much.

**Comment by youknowone** [comment]
I like this feature. I want to discuss about user experience. `Say` is a light-weight utility. For most of user, `Rate` and `0.5` is too much scientistic and not understandable. How about just using `Slow` and `Fast`?

**Comment by Amanida** [comment]
@youknowone That's cool! I'll do it.

**Pull request merged**